vmcnet/utils/io.py

```diff
@@ -34,6 +34,26 @@ def append_metric_to_file(new_metric, logdir, name):
         np.savetxt(outfile, dumped_metric)
 
 
+def copy_txt_stats(source_dir, target_dir, truncate=None):
+    """Copies and truncated energy.txt as similar stats."""
+    names = ["energy", "energy_noclip", "variance", "variance_noclip", "accept_ratio"]
```

Comment:
This is cool, the only part I don't like is that it's these 5 metrics only whereas there are options like `record_amplitudes` that result in other log files being made. I think we could handle this if we push the call to `copy_txt_stats` into the `vmc.py` `vmc_loop` function. Then we could do one iteration of the `update_param_fn`, get back a `metrics` object, and use the keys of the `metrics` dict to decide what data to copy over. It would probably add a little bit of extra threading of arguments around but it might be worth it to make the functionality more generic. What do you think?

Reply:
changed to move all txt files except git_hash